.fixtures.yml

```diff
@@ -1,5 +1,3 @@
 fixtures:
-  repositories:
-    stdlib: 'http://github.com/puppetlabs/puppetlabs-stdlib.git'
```

Comment:
Fairly unusually, this module doesn't actually use any of stdlib (and it's not a metadata.json dependency).  No point cloning it for nothing.